cluster/gce/util.sh

```diff
@@ -810,17 +853,20 @@ function kube-up {
 # down the firewall rules and routes.
 function kube-down {
   detect-project
+  detect-node-names # For INSTANCE_GROUPS
 
   echo "Bringing down cluster"
   set +e  # Do not stop on error
 
-  # Delete autoscaler for nodes if present.
+  # Delete autoscaler for nodes if present. We assume that all or none instance groups have an autoscaler
   local autoscaler
   autoscaler=( $(gcloud compute instance-groups managed list --zone "${ZONE}" --project "${PROJECT}" \
-                 | grep "${NODE_INSTANCE_PREFIX}-group" \
+                 | grep "${NODE_INSTANCE_PREFIX}-group-1" \
```

Comment:
a -1 group may not exist (with less than 500 nodes it won't) so that should check "${NODE_INSTANCE_PREFIX}-group"